docs/getting-started-auditor.md

```diff
@@ -0,0 +1,170 @@
+Note that you also need to enable `scalar.dl.ledger.proof.enabled` and set a proper private key to `scalar.dl.ledger.proof.private_key_path` since Auditor uses [AssetProof](how-to-use-proof.md) to work.
+If they are not properly and consistently configured, Ledger will throw an exception.
+
+Also, Ledger manages an Auditor's certificate (i.e., Auditor needs to register its certificate to Ledger) and uses the certificate to validate a request signed by Auditor's private key to identify the request origin; thus, the certificate holder ID (`scalar.dl.ledger.auditor.cert_holder_id`) and version (`scalar.dl.ledger.auditor.cert_version`) are required to be set as well. 
+By default, Ledger assumes Auditor registers its certificate with a name `auditor` and version `1`.
+
+For the properties of Auditor, you need to configure the following entries:
+```
+[auditor.properties]
+#scalar.dl.auditor.ledger.cert_holder_id=ledger
+#scalar.dl.auditor.ledger.cert_version=1
+#scalar.dl.auditor.cert_holder_id=auditor
+#scalar.dl.auditor.cert_version=1
+scalar.dl.auditor.cert_path=/path/to/auditor.pem
+scalar.dl.auditor.private_key_path=/path/to/auditor-key.pem
+```
+
+Auditor manages a key pair to sign a request before sending the request to Ledger and validate a request given from Ledger,
+so you need to set `scalar.dl.auditor.cert_path` and `scalar.dl.auditor.private_key_path` properly.
+
+As similarly to Ledger, Auditor manages a Ledger's certificate (i.e., Ledger registers its certificate to Auditor) and uses the certificate to validate a request signed by Ledger's private key; thus, the certificate holder ID (`scalar.dl.auditor.ledger.cert_holder_id=ledger`) and version (`scalar.dl.auditor.ledger.cert_version`) are required.
+By default, Auditor assumes Ledger registers its certificate with a name `ledger` and version `1`.
+
+Other values are optional when you but they need to be updated depending on an environment.
+For example, the default settings assume that Ledger is placed in the same host, and Auditor uses locally installed Cassandra through Scalar DB without configuring the password.
+Please check [the configuration file](https://github.com/scalar-labs/scalar/blob/master/auditor/conf/auditor.properties.tmpl) for more detail.
+
+
+## Start Ledger and Auditor
+
+Please start Ledger and Auditor in your own way.
+For example, if you use the built-in command line tools, do as follows:
+
+Ledger:
+```
+bin/scalar-ledger --properties ledger.properties
+```
+
+Auditor:
+```
+bin/scalar-auditor --properties auditor.properties
+```
+
+You can also use the containers that are available in [the GitHub Container Registry](https://github.com/orgs/scalar-labs/packages).
+See [saclardl-samples](https://github.com/scalar-labs/scalardl-samples) repo for more details about how to use the containers.
+
+## Register each certificates of Ledger and Auditor
```

Comment:
Each is used for singular words so it should be "each certificate" or "all certificates" if the meaning needs to be plural.

Reply:
Thank you!
Fixed in dc5c4c4f08129b03b2458e5a3de1279510e84f10.